### PR TITLE
Responsive home page layout

### PR DESCRIPTION
**Pull request by Bravax**

### CODE REVIEW

File: frontend/src/components/LevelPage/GameMenu/Inventory/Inventory.jsx [hunk 1]
```diff
@@ -25,13 +24,12 @@ function Inventory() {
...
       >
         <section className="openBag">
-          <img src={OpenBag} alt="Open Bag" className="insideBag" />
           <button
             type="button"
             className="inventoryPopUp__closeButton"
             onClick={handleInventoryToggle}
           >
-            Fermé
+            X
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/GameMenu/Inventory/Inventory.jsx:32]: An aria-label is needed to be more explicit about the button's function.

File: frontend/src/components/LevelPage/GameMenu/Objectives/Objectives.jsx [hunk 2]
```diff
@@ -24,20 +23,13 @@ function Objectives() {
...
-          >
-            Fermé
-          </button>
-        </section>
+        <button
+          type="button"
+          className="objectivesPopUp__closeButton"
+          onClick={handleObjectivesToggle}
+        >
+          X
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/GameMenu/Objectives/Objectives.jsx:31]: An aria-label is needed to be more explicit about the button's function.

File: frontend/src/components/LevelPage/TargetButton/TargetButton.jsx [hunk 3]
```diff
@@ -1,58 +1,77 @@
...
+                    aria-label={`${item.itemName}`}
+                  >
+                    <img src={item.itemSrc} alt="" className={item.itemClass} />
+                  </button>
+                ) : (
+                  ""
+                )}
+                <h2 className="name">{dino.name}</h2>
+              </header>
+              <main>
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/TargetButton/TargetButton.jsx:32]: Incorrect use of the main tag

File: frontend/src/components/LevelPage/TargetButton/TargetButton.scss [hunk 4]
```diff
@@ -19,61 +46,59 @@
...
   flex: 1;
-  border-left: 0.125rem solid $secondaryColor;
-  border-right: 0.125rem solid $secondaryColor;
+  border-left: 2px solid $secondaryColor;
+  border-right: 2px solid $secondaryColor;
   height: 1.5vh;
-  margin-right: 1.2rem;
-  margin-left: 0.6rem;
-  padding-left: 0.3rem;
+  margin-right: 20px;
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/TargetButton/TargetButton.scss:90]: Responsive values for margins and padding please

**Review by Yoann-Fortin — CHANGES_REQUESTED** [review]

File: frontend/src/components/LevelPage/GameMenu/Objectives/Objectives.jsx (hunk 2 above)
Comment (Bravax) [reply frontend/src/components/LevelPage/GameMenu/Objectives/Objectives.jsx:31]: Corrected

File: frontend/src/components/LevelPage/GameMenu/Inventory/Inventory.jsx (hunk 1 above)
Comment (Bravax) [reply frontend/src/components/LevelPage/GameMenu/Inventory/Inventory.jsx:32]: Corrected

File: frontend/src/components/LevelPage/TargetButton/TargetButton.jsx (hunk 3 above)
Comment (Bravax) [reply frontend/src/components/LevelPage/TargetButton/TargetButton.jsx:32]: Replace with a footer

File: frontend/src/components/LevelPage/TargetButton/TargetButton.scss (hunk 4 above)
Comment (Bravax) [reply frontend/src/components/LevelPage/TargetButton/TargetButton.scss:90]: Modified

**Pull request closed without merging**